nepalingo-web/src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+import useDictionary, { DictionaryProps, DictionaryResponse } from '../../hooks/useDictionary';
```

Comment:
Looks lije there might be some lint errors

also instead of going up using `../../`, it would be easier if you went down - does `./hooks/useDictionary` work?